mythril/laser/ethereum/evm_exceptions.py

```diff
@@ -6,5 +6,8 @@ class StackUnderflowException(VmException):
     pass
 
 
-class StopSignal(Exception):
+class StackOverflowException(VmException):
```

Comment:
Can maybe make this:
```
StackOverflowException(VmException, IndexError)
```
That way the analysis modules and svm.py won't have to depend on the StackOverflowException, but can just keep on working with IndexErrors